src/components/SidebarGroup.vue

```diff
@@ -20,9 +20,9 @@
     user-select: none;
     color: var(--sidebar-title-color);
     line-height: 1;
-    font-size: 14px;
+    font-size: 12px;
```

Comment:
I would go against 12px in general, since it's really small on screens with higher resolutions (considering 1920x1080 is the standard)

Reply:
![](https://i.imgur.com/YTc7bEX.png)
I am working on a 2.5k res and based on the screenshot there is not too much difference. Imo it's a sub heading which is easy to read, in that screenshot i added a letterspacing w/ 1px to the subheadings for better readability.

Reply:
We could also use 14px and then apply a lighter font weight:
![](https://i.imgur.com/ZIsQqIA.png)

Reply:
yeah push with 14px and less font weight and I'll review and accept